Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureWebRole/TypeScriptWebApp.njsproj

```diff
@@ -12,7 +12,6 @@
     <ProjectGuid>$guid1$</ProjectGuid>
     <ProjectHome>.</ProjectHome>
     <StartupFile>server.ts</StartupFile>
-    <StartWebBrowser>True</StartWebBrowser>
```

Comment:
these seem like local changes

Reply:
Turns out it isn't! This property is duplicated in some files for some reason. You can see another one in line 23 if you expand the file! This was breaking me for picking up in C# but it was working on MSBuild because MSBuild will pick the last value it computes (if there's two).